### ISSUE TITLE
Speed limit as rule's road, not as device parameter

### ISSUE THREAD

**Issue by ghost** [vitalidze/traccar-web#343]
Hello,

While using my gps navigator, i've sounds notification when I overspeed.

Is it possible to have some overspeed notification as the road's rules, not as device's parameter ?

**Comment by vitalidze** [comment]
No, it's not possible. This requires access to some speed limit database. I have no idea how to implement it in an easy and a lightweight manner.

**Comment by ghost** [comment]
I understand, as a different way, perhaps using geofence polygon as a  x speed limit inside ????

**Comment by vitalidze** [comment]
I will think about it.

**Comment by MMartinezV** [comment]
Hi,

I agree that It would be nice to have geofences with a defined speed limit that overrides the device's speed.

Some time ago, I read about how to query OpenStreetMap and get the type of a way (highway={primary, secondary, tertiary, residential, service, ...}) and it has also defined an optional tag "maxspeed" (numeric?).

For example, if we look at http://www.openstreetmap.org/way/14519854, we can see the Mallorcan road MA-15 with speed limit=100 and highway=primary.

More info at: 

http://wiki.openstreetmap.org/wiki/Key:maxspeed
http://wiki.openstreetmap.org/wiki/Nominatim#Special_Keywords

Anyway, I think it would be easier and very useful to be able to assign a speed limit to a Geofence.

Regards,

  Manuel
